pkg/sfu/receiver.go

```diff
@@ -360,52 +334,80 @@ func (w *WebRTCReceiver) writeRTP(layer int) {
 			return
 		}
 
-		w.locks[layer].Lock()
+		if w.isSimulcast {
+			pts := w.pendingTracks[layer].Load().([]*DownTrack)
+			if len(pts) > 0 {
+				if pkt.KeyFrame {
+					w.Lock()
+					for _, dt := range pts {
```

Comment:
I think we'll need to reload pts here in order to avoid the race.

Reply:
Agree there is super small probability that during the first Load and get the lock something modifies the slice.